app/views/components/language_selection.scala.html

```diff
@@ -1,31 +0,0 @@
-@*
- * Copyright 2022 HM Revenue & Customs
- *
```

Comment:
why was this file deleted?

Reply:
it wasn't being used and all it was basically a wrapper for the now deprecated uk.gov.hmrc.urls.Link component